<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SaveNotesDialog</class>
 <widget class="QDialog" name="SaveNotesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>240</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>380</width>
    <height>240</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>380</width>
    <height>240</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Save notes preset as...</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {background-color:#333;}
QWidget {color:#ddd;}

QComboBox {background-color:#4A4A4A;}
QComboBox QAbstractItemView {background-color:#444;}
QComboBox:hover{background-color:#405080;}
QComboBox:!enabled{background-color:#202020;color:#808080;}

QPushButton{background-color:#444;color:#EEE;border:1px solid #888; border-radius:4px;}
QPushButton:!enabled{background-color:#202020;color:#808080}
QPushButton:hover{background-color:#405080;}
QPushButton:pressed {background:#505020;border:1px inset #222;}
QSpinBox:!enabled {color:#777;}
QDoubleSpinBox:!enabled {color:#777;}
QLabel {background-color: rgba(0, 0, 0, 0);}
QLabel:!enabled {color:#888;}
QLineEdit {background-color:#333;color:#EEE;border:1px solid #888;}
QLineEdit:hover{background-color:#446;}
QLineEdit:!enabled {background-color:#202020;color:#888;}
QMessageBox QPushButton {width:80px;height:20px;}</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <widget class="QLineEdit" name="nameLineEdit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>150</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="maxLength">
    <number>40</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="savePushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>1</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Save notes preset</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelPushButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>200</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>1</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="nameLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Custom Label (optional)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="saveListLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Select destination</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="saveListComboBox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>110</y>
     <width>201</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="maxVisibleItems">
    <number>15</number>
   </property>
  </widget>
  <widget class="QRadioButton" name="user1Radio">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>84</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>User1</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="user2Radio">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>50</y>
     <width>84</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>User2</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="user3Radio">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>80</y>
     <width>84</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>User3</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="custRadio">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>110</y>
     <width>16</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
